graphql/src/adapters/graphql/dataTypes/echo.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+# noinspection PyPackageRequirements
+import strawberry
+
+from src.entities.echo import Echo
+
+
+# todo: While it technically breaks CleanArchitecture,
+#  I wonder if we wouldn't be fine placing these decorators on the base class
+#  given that they don't change coding behavior.
```

Comment:
Main question for team here. If we add this decorator to the `entities` package, we won't have to duplicate every class here. That said, it does break clean architecture.

Reply:
we should first decide if we want to keep clean architecture for now.